maps/away/scenarios/enviro_testing_facility/enviro_testing_facility.dm

```diff
@@ -0,0 +1,118 @@
+
+// ---- map template/archetype
+
+/datum/map_template/ruin/away_site/enviro_testing_facility
+	name = "Environmental Testing Facility"
+	description = "Environmental Testing Facility."
+	id = "enviro_testing_facility"
+
+	prefix = "scenarios/enviro_testing_facility/"
+	suffix = "enviro_testing_facility.dmm"
+
+	exoplanet_theme_base = /datum/exoplanet_theme/barren
+	exoplanet_themes = list(
+		/turf/unsimulated/marker/blue = /datum/exoplanet_theme/barren,
+		/turf/unsimulated/marker/red   = /datum/exoplanet_theme/barren/mountain,
+	)
+	exoplanet_atmospheres = list(/datum/gas_mixture/earth_standard)
+	exoplanet_lightlevel = list(1, 2, 5, 7)
+	exoplanet_lightcolor = list("#ffffd4", "#ffe397", "#b38653")
+
+	spawn_cost = 1
+	spawn_weight = 0 // so it does not spawn as ordinary away site
+	sectors = list(ALL_POSSIBLE_SECTORS)
+	unit_test_groups = list(1)
+
+/singleton/submap_archetype/enviro_testing_facility
+	map = "Environmental Testing Facility"
+	descriptor = "Environmental Testing Facility."
+
+// --------------------------------------------------- sector
+
+/obj/effect/overmap/visitable/sector/enviro_testing_facility
+	name = "Anastasia-Archippos, Barren Planet"
+	desc = "\
+		Barren planet, covered in loose rocks, mountains, craters, with a expansive cave system deep underground. \
+		The planet, however, holds a standard breathable atmosphere, and there are some traces of carbon-based life. \
+		Landing site is in a valley, near some facility carved into the mountain.\
+		"
```

Comment:
https://github.com/Aurorastation/Aurora.3/blob/master/.github/guides/STYLE.md#text-variables

`Do not leave empty lines just to isolate the text, the text should start right after the " and end right before the ".`

Reply:
https://github.com/Aurorastation/Aurora.3/pull/20514